Libs/MRML/Core/vtkDataFileFormatHelper.cxx

```diff
@@ -55,6 +55,7 @@ ITKImageFileFormat FileFormatTable[] =
   {"MGHImageIO", "Uncompressed FreeSurfer pixel data in binary", "MGH", ".mgh"},
   {"MGHImageIO", "Gzip Compressed FreeSurfer pixel data in binary", "MGH", ".mgz"},
   {"MGHImageIO", "Gzip Compressed FreeSurfer pixel data in binary", "MGH", ".mgh.gz"},
+  {"MINCImageIO", "Optional compression. Binary pixel data", "MINC", ".mnc"},
```

Comment:
Integrated MINC ITK reader is based only on MINC2 library, and is not capable of reading MINC1 files. 
If one would compile full minc library (with MINC1 & MINC2 support) and incorporated it into Slicer, then ITK would be able to read .mnc.gz files.